fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -57,22 +57,56 @@ const EventRollCall = (props: IPropTypes) => {
         );
         return;
       }
-      requestOpenRollCall(event.idAlias).catch((e) =>
-        console.debug('Unable to send Roll call re-open request', e),
-      );
+      const time = Timestamp.EpochNow();
+      requestReopenRollCall(event.idAlias, time)
+        .then(() => {
+          if (!event.idAlias) {
+            console.debug(
+              'Unable to send roll call re-open request, the event does not have an idAlias',
+            );
+            return;
+          }
+          navigation.navigate(STRINGS.roll_call_open, {
+            rollCallID: event.id.toString(),
+            updateID: Hash.fromStringArray(
+              EventTags.ROLL_CALL,
+              lao.id.toString(),
+              event.idAlias.toString(),
+              time.toString(),
+            ),
+          });
+        })
+        .catch((e) => console.debug('Unable to send Roll call re-open request', e));
     } else {
       const time = Timestamp.EpochNow();
       requestOpenRollCall(event.id, time)
         .then(() => {
           navigation.navigate(STRINGS.roll_call_open, {
             rollCallID: event.id.toString(),
-            time: time.toString(),
+            updateID: Hash.fromStringArray(
+              EventTags.ROLL_CALL,
+              lao.id.toString(),
+              event.id.toString(),
+              time.toString(),
+            ),
           });
         })
         .catch((e) => console.debug('Unable to send Roll call open request', e));
     }
   };
 
+  // Scanning attendees should be available only when the Roll Call is in state opened or reopened => idAlias is defined
+  const onScanAttendees = () => {
+    if (!event.idAlias) {
+      console.debug('Unable to scan attendees, the event does not have an idAlias');
+      return;
+    }
```

Comment:
are you trying to assert a case that should always, absolutely be true ? 
If so, you might want to make this error as loud as possible, e.g. `console.error` or even better throw an exception !

Reply:
Turned it into a console.error